crawl-ref/source/spl-summoning.cc

```diff
@@ -817,13 +817,16 @@ int mons_ball_lightning_hd(int pow, bool random)
     return ball_lightning_hd(pow, random) / 2;
 }
 
-spret cast_conjure_ball_lightning(int pow, god_type god, bool fail)
+spret cast_conjure_ball_lightning(const actor &agent, int pow, god_type god, bool fail)
 {
     fail_check();
     bool success = false;
 
-    mgen_data cbl =_pal_data(MONS_BALL_LIGHTNING, 0, god,
-                             SPELL_CONJURE_BALL_LIGHTNING);
+    const auto att = agent.is_player() ? BEH_FRIENDLY
+                                       : SAME_ATTITUDE(agent.as_monster());
```

Comment:
I believe you can just use `BEH_COPY` here, as `_summon_data()` does.

Reply:
thanks, will also mark as resolved, since [suggestion from last converstation](https://github.com/crawl/crawl/pull/2813#discussion_r1063960530) fixes it.